libnavui-maps/src/test/resources/artificially_wrong_two_leg_route.json

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
nit: if you remove the white space from these, it reduces the size by a lot.

Before: 46KB
After: 19KB

KB is small. But percentage wise, size reduction of 242%

Reply:
I"ll minify them.